engine/src/main/java/org/terasology/engine/modes/loadProcesses/CreateWorldEntity.java

```diff
@@ -52,64 +55,101 @@ public String getMessage() {
 
     @Override
     public boolean step() {
+        if (worldEntityExists()) {
+            EntityRef worldEntity = getExistingWorldEntity();
+            configureChunkProvider(worldEntity);
+            useConfigurationOfCurrentWorld(worldEntity);
+        } else {
+            EntityRef worldEntity = createWorldPoolsAndEntity();
+            configureChunkProvider(worldEntity);
```

Comment:
is it important that configureChunkProvider happens before we do other stuff with the WorldConfigurator, or may we move it down out of this branch to the common path?